dd-sdk-android/src/main/kotlin/com/datadog/android/v2/core/NoOpSdkCore.kt

```diff
@@ -21,7 +22,7 @@ import java.util.concurrent.TimeUnit
 /**
  * A no-op implementation of [SdkCore].
  */
-internal class NoOpSdkCore : SdkCore {
+internal class NoOpSdkCore : FeatureSdkCore {
```

Comment:
NitPick, I'd rename this as `NoOpFeatureSdkCore`

Reply:
indeed, makes sense. I will do it in another PR, to avoid rebasing other stacked PRs.